generate-stackbrew-pr.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+set -e
+. functions.sh
+
+GITHUB_USERNAME="nodejs-docker-bot"
```

Comment:
`nodejs-github-bot` as concluded in https://github.com/nodejs/docker-node/issues/670#issuecomment-378195959, for the record I've forked the upstream official-images repo to that GitHub account.